widget/embedded/src/components/HeaderButtons/InProgressTransactionBadge.tsx

```diff
@@ -0,0 +1,32 @@
+import type { PendingSwap } from 'rango-types';
+
+import { useManager } from '@rango-dev/queue-manager-react';
+import { InProgressIcon, styled } from '@rango-dev/ui';
+import React from 'react';
+
+import { getPendingSwaps } from '../../utils/queue';
+
+export const Container = styled('div', {
```

Comment:
I think it's better to have all the styles in the **HeaderButtons.styles.ts**

Reply:
fixed